spec/lib/blacklight/solr/search_builder_spec.rb

```diff
@@ -553,7 +553,71 @@
       end
     end
   end
-  
+
+  describe "#add_facet_paging_to_solr" do
+    let(:facet_field) { 'format' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.